staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go

```diff
@@ -94,7 +94,7 @@ var ErrorHandlers = []func(error){
 	}).OnError,
 }
 
-// HandlerError is a method to invoke when a non-user facing piece of code cannot
+// HandleError is a method to invoke when a non-user facing piece of code cannot
 // return an error and needs to indicate it has been ignored. Invoking this method
```

Comment:
I think `to indicate it has been ignored` might be confusing. `to indicate that the/this error has been ignored` could be clearer. 
We can change it as well. @liggitt WDYT?

Reply:
Let me re-phrase it.